### PR TITLE
Copied the work from the fork repo to the original repo in order to u…

### PR DESCRIPTION
**Pull request by hungEA**
**Title:** [248 - Develop data validator](https://www.pivotaltracker.com/story/show/184613248)

**Description:**

The data-validator program will be able to run validation check for the required csv files that would be push to WRGL before being imported to BE database. The process will be applied to the following data

- [x] agency_reference_list
- [x] personnel
- [x] allegation
- [x] appeals
- [x] use_of_force
- [x] citizens
- [x] event
- [x] person
- [x] documents

### CODE REVIEW

**Comment by baoea** [comment]
Hi @ayyubibrahimi. Could you explain us why this process fails? We need to pass this step to test our validator. Thanks!

**Comment by ayyubibrahimi** [comment]
I'll review.

**Comment by ayyubibrahimi** [comment]
I will re-write this function.

**Comment by ayyubibrahimi** [comment]
These bugs are weird. I can't reproduce them on my local when I run `make`, `make` ran fine with the process-data step for my last PR (the PR that you just merged into this branch), and no arguments are being passed to the similarity function. Can you please advise @pckhoi.

File: requirements.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 numpy==1.23.4
 pandas==1.4.2
-python-Levenshtein==0.12.0
+levenshtein==0.20.9
```
Comment (pckhoi) [inline requirements.txt:3]: @ayyubibrahimi this is the culprit. `datamatch` is dependent on python-Levenshtein v0.12.0. I wonder why this dependency is updated in the first place.

Comment (hungEA) [reply requirements.txt:3]: If we don't update that dependency, we can't build our docker image. Have you recently tried to rebuild the docker image with `python-Levenshtein==0.12.0`?

Comment (pckhoi) [reply requirements.txt:3]: Actually they seem to be 2 separate packages, so they can probably coexist. Can you try putting back in the python-levenshtein package?

Comment (pckhoi) [reply requirements.txt:3]: If that doesn't work I'll update datamatch. It should work because both packages seem to be just different versions.

Comment (ayyubibrahimi) [reply requirements.txt:3]: Thanks Khoi. I'm not near my laptop atm but I'll update it ASAP.

Comment (hungEA) [reply requirements.txt]: I reverted it back to the original version. I also had to clean up the cache in the docker builder. It's successfully built the docker image.

**Comment by hungEA** [comment]
@ayyubibrahimi @pckhoi our data-validator processes have successfully passed. However, we are not sure why the `commit changes in transaction` step failed as we did not touch this section at all. Can you have a look please?

**Comment by ayyubibrahimi** [comment]
Awesome. From what I understand this is a issue with the server connection. Please re-run the failed jobs and it should eventually process successfully.

**Pull request merged**